workflows/rnaseq/main.nf

```diff
@@ -299,10 +299,15 @@ workflow RNASEQ {
 
     FASTQ_SUBSAMPLE_FQ_SALMON
         .out
-        .json_info
+        .lib_format_counts
         .join(ch_strand_fastq.auto_strand)
         .map { meta, json, reads ->
-            return [ meta + [ strandedness: getSalmonInferredStrandedness(json) ], reads ]
+            def salmon_strand_analysis = getSalmonInferredStrandedness(json, threshold = params.strand_predict_threshold)
```

Comment:
I'd rather we don't use params in the workflows, but pass over the value from when we call the workflow in the main `main.nf`

Reply:
There are 'params' usages all over this workflow file. I agree with the principle, but we should fix that in a different PR I think, not use a different system for this PR